archive/TroubleshootRsv

```diff
@@ -0,0 +1,110 @@
+---+!! !RSV Troubleshooting Guide
+%DOC_STATUS_TABLE%
+%TOC%
+
+---+ About this Document
+This document describes  how to troubleshoot !RSV v3 (part of OSG 3.0).
+
+---+ How to get Help?
+To get assistance please use [[HelpProcedure][Help Procedure]].
+The developers of a specific metric/probe may provide additional support channel, please check the specific troubleshooting document if your metric is listed in the table below.
+
+%INCLUDE{"RsvControl" section="RsvProfilerNT"}%
```

Comment:
Could you take everything from this line up until the reference section, convert it, and add it to the troubleshooting section of the RSV doc [here](https://opensciencegrid.github.io/docs/monitoring/rsv/#troubleshooting-rsv)? Don't forget the contents of the `%INCLUDE`

Reply:
@edquist whenever you get to the above and submit the PR, link to it here, and we can move forward on getting this PR wrapped up.

Reply:
Since we're archiving all the docs linked in the table, this doc is less useful. Pushing through with the archival